src/components/SearchForm/SearchForm.js

```diff
@@ -1,16 +1,26 @@
-import React from "react";
 import "./SearchForm.css";
 
-export default function SearchForm() {
+export default function SearchForm({ onSearch, keyword, setKeyword }) {
+
+  function handleChange(e) {
+    setKeyword(e.target.value)
+  }
+
   return (
     <div className="search-form">
       <div className="search-form__container">
-        <form className="search-form__container-form">
+        <form
+          className="search-form__container-form"
+          onSubmit={(e) => onSearch(e, keyword)}
+        >
           <label className="search-form__container-form-field">
             <input
               className="search-form__input"
               type="text"
               placeholder="Enter Topic"
+              autoFocus
+              autoComplete="off"
+              onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components